src/backend/distributed/sql/citus--10.1-1--10.2-1.sql

```diff
@@ -16,6 +16,8 @@ ALTER TABLE pg_catalog.pg_dist_placement ADD CONSTRAINT placement_shardid_groupi
 #include "udfs/citus_internal_update_placement_metadata/10.2-1.sql";
 #include "udfs/citus_internal_delete_shard_metadata/10.2-1.sql";
 #include "udfs/citus_internal_update_relation_colocation/10.2-1.sql";
+#include "udfs/create_time_partitions/10.2-1.sql"
+#include "udfs/get_missing_time_partition_ranges/10.2-1.sql"
```

Comment:
Open question: Is it worth introducing the timeseries directory in this PR?

Reply:
I don't want to add timeseries directory to not to cause a confusion. If we will continue workload, we can create it.